Sources/Mantis/Helpers/TransformStack.swift

```diff
@@ -0,0 +1,57 @@
+import UIKit
+
+
+class TransformStack: NSObject {
+    
+    static var shared: TransformStack = TransformStack()
+    
+    weak var transformDelegate : TransformDelegate!
+    
+    var transformAdjustmentsStack : Array = [TransformRecord]()
+    
+    var isRotated : Bool! = false
+    
+    var bottom : Int = 0
+    var top : Int = 0
+    
+    var transformStackBottom : Int = 0
+    var transformStackTop : Int = 0
+        
```

Comment:
I suggest adding private to these properties when possible and remove unnecessary `!`

```Swift
    private var transformAdjustmentsStack: [TransformRecord] = []
    
    private var isRotated: Bool = false
    
    private var bottom: Int = 0
    
    var top: Int = 0
    
    private var transformStackBottom: Int = 0
    private var transformStackTop: Int = 0
···

Reply:
changes made